### PR TITLE
[MDS-102] Add TEPP events from excel file

### PR DESCRIPTION
**Pull request by ddhogan**

### CODE REVIEW

**Comment by loren** [comment]
@ddhogan If you want, feel free to add me as a reviewer when this PR is ready. Looks great so far!

File: service.py [hunk 1]
```diff
@@ -105,3 +112,85 @@ def get_soup(link):
...
+
+
+"""
+Access the pre-processed Excel file from the s3 bucket.
+Generate a json array of events from the Excel file.
+Starting with the list of ITA events, concatenate the TEPP events.
+Upload the concatenated list to the s3 bucket.
+"""
+def get_tepp_worksheet():
+  s3_response_object = s3.get_object(Bucket='trade-events', Key='tepp_prepared.xlsx')
```
Comment (loren) [inline service.py]: maybe extract "trade-events" string as a constant?

File: service.py [hunk 2]
```diff
@@ -4,23 +4,30 @@
...
-def handler(event, context):
-    entries = get_event_list()
-    if len(entries) > 0:
-        s3.Object('trade-events', 'ita.json').put(Body=json.dumps(entries), ContentType='application/json')
-        return "Uploaded ita.json file with %i trade events" % len(entries)
-    else:
-        return "No entries loaded from %s so there is no JSON file to upload" % ENDPOINT
+def handler(event, context):  
+    entries = get_concat_events()
+    try:
```
Comment (loren) [inline service.py]: This is a much better approach than the way I've been doing it in the other lambdas. Nice!

File: service.py (hunk 1 above)
Comment (ddhogan) [reply service.py]: Sounds good, I'll push a commit for this

File: service.py (hunk 2 above)
Comment (ddhogan) [reply service.py]: thanks! =]

**Pull request merged**